ironfish/src/account/account.ts

```diff
@@ -22,10 +22,14 @@ export class Account {
     Readonly<{
       transaction: Transaction
       blockHash: string | null
+      sequence: number | null
```

Comment:
should this be blockSequence to match that it's the sequence that came from the block its on?